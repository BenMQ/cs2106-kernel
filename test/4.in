cr a 2
cr a 1

init
cr a 2
cr b 0

init
cr a 2
req R2 2
cr b 2
to
req R2 1
rel R2 1
to
de a
