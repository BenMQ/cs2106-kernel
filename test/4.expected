init a error 
init a error 
init a a a b a a b error 
